### ISSUE TITLE
PDF Export

### ISSUE THREAD

**Issue by rjmoggach** [django-wiki/django-wiki#101]
_File this under enhancements_

It would be great to support hyperlinked PDF export of pages and corresponding print media css.

**Comment by jdcaballerov** [comment]
Hi, I think this enhancement is not very difficult but a dependency must be used. I had a great experience using weasyprint for another project.

**Comment by benjaoming** [comment]
Agreed... but seeing that articles are sorted into hierarchies and has different permissions, perhaps it would be useful to have a UI where one can select and sort the articles to be included in the PDF rather than just having "export this article as PDF".

Well, the button could still be the same, but the UI could look at `article.get_children()` and present all children with check boxes and a sorting options.

**Comment by jdcaballerov** [comment]
A thing to take into account in that case of multiple pages is that producing the pdf might take some time and the process is blocking. I use celery to offload but I guess here is overkill.

**Comment by benjaoming** [comment]
I think you're right that this is too much to have a view function handle. It's also on the TODO for django-nyt to have an option to use a task queue like Celery.
